1-exercises/B-todo-list/script.js

```diff
@@ -1,12 +1,26 @@
 function todoList(todos) {
   // Write your code here...
+  const UL = document.createElement("ul");
+  document.getElementById("content").append(UL);
+  for (let todo of todos) {
+    let li = document.createElement("li");
+    li.innerText = `${todo.todo}`;
+    li.addEventListener("click", () => {
+      if (li.classList.contains("done")) {
+        li.classList.remove("done");
+      } else {
+        li.classList.add("done");
+      }
+    });
```

Comment:
The code is functionally correct, but it would have been better to abstract this arrow function into its own named function instead.